### ISSUE TITLE
Suggestion: Allow pool of already instantiated browser workers

### ISSUE THREAD

**Issue by tehandyb** [thomasdondorf/puppeteer-cluster#524]
Would it be possible to have browser workers already available instead of launching the browser when a new browser worker instance is created? This would allow incoming calls that need a browser, to immediately get it, instead of having to wait for the browser to startup. Similar to how the Page based worker already launches the browser in its init() function. I guess the logic might get a bit complicated on keeping the browser instances around, but I think it could be worth it

**Comment by cognettings** [comment]
Yeah this would be nice. There's a cool npm package (super outdated though) that creates a pool of puppeteer browsers and reuses them between requests. The pool can be configured to refresh the browsers after they handle a given number of requests (like the Apache server process pool). [puppeteer-pool](https://www.npmjs.com/package/puppeteer-pool)
